sematic/examples/testing_pipeline/__main__.py

```diff
@@ -92,11 +92,7 @@
     "KubernetesResourceRequirements configuration containing all the relevant specified "
     "CLI parameters. Defaults to False."
 )
-RAY_HELP = (
-    "Includes a function that is executed on the specified external Ray cluster. "
-    "using a remote Ray cluster. If not provided, Ray will not be used. "
-    "Example: 'ray://raycluster-complete-head-svc:10001'."
-)
+RAY_HELP = "Includes a function that is executed on an external Ray cluster. "
```

Comment:
We will probably need an `ee` testing pipeline to keep things clean. Another TODO...

Reply:
we decided to keep this in the normal testing pipeline